### PR TITLE
private browser test

### PR DESCRIPTION
**Pull request by Daisy-pliu**
1. [TC0108] Open private mode and back when browser tab number is 0
2. [TC 0109] Open private mode and back when browser tab number is over 0.

### CODE REVIEW

File: app/src/androidTest/java/org/mozilla/focus/helper/RecyclerViewMatcher.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+import android.content.res.Resources;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+public class RecyclerViewMatcher {
```
Comment (cnevinc) [inline app/src/androidTest/java/org/mozilla/focus/helper/RecyclerViewMatcher.java]: we already have this https://github.com/mozilla-tw/FirefoxLite/blob/58f460ab11502e3598bf751e518494c9ea69dfba/app/src/androidTest/java/org/mozilla/focus/utils/RecyclerViewTestUtils.java#L27

**Comment by mTwTm** [comment]
Please rebase to current firefoxlite/master to get rid of a test intermittent.

**Comment by Daisy-pliu** [comment]
rebased

File: app/src/androidTest/java/org/mozilla/focus/utils/AndroidTestUtils.java [hunk 2]
```diff
@@ -132,6 +132,14 @@ public static void tapSettingButton() {
         onView(allOf(withId(R.id.menu_preferences), isDisplayed())).perform(click());
     }
 
+    public static void tapPrivateButtonInMenu() {
+        onView(allOf(withId(R.id.btn_private_browsing), isDisplayed())).perform(click());
```
Comment (cnevinc) [inline app/src/androidTest/java/org/mozilla/focus/utils/AndroidTestUtils.java]: there are two view with the same id : btn_private_browsing
not sure if that's the cause

**Pull request closed without merging**